atlp-rwanda/elite-bn-be
# Ft-google-facebook-auth-182201570

**What does this PR do?**
Allow users to log in with their Google and Facebook

**How should this be manually tested?**

1. Create Google Client and Secret IDs: you can follow this [tutorial](https://www.youtube.com/watch?v=xH6hAW3EqLk) for guidance.

2. Create Facebook Client and secret IDs. You can follow this [tutorial](https://www.youtube.com/watch?v=jMVlSDU-6l4) to do so.

3. Follow the.env.example file to create your .env file and add your credentials

4. Run migration
5. Start the server

Check these endpoints:

`GET : auth/google`
`GET : auth/facebook`

**Any background context you want to provide?**

You need to have both the endpoint path and a correct callback_url to successfully test this PR.

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/182201570
